internal/runners/clean/run_lin_mac.go

```diff
@@ -127,6 +135,25 @@ func removeInstall(cfg *config.Instance) error {
 	return aggErr
 }
 
+func removeApp() error {
+	svcExec, err := installation.ServiceExec()
+	if err != nil {
+		return locale.WrapError(err, "err_svc_exec")
+	}
+
+	svcApp, err := app.New(constants.SvcAppName, svcExec, []string{"start"}, svcApp.Options)
+	if err != nil {
+		return locale.WrapError(err, "err_autostart_app")
+	}
```

Comment:
Seems this has to stay in sync with the install step or this will break, maybe centralize this somewhere?